src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/FilterFunctions_V1Compat.txt

```diff
@@ -0,0 +1,5 @@
+#SETUP: TableSyntaxDoesntWrapRecords,PowerFxV1CompatibilityRules
+
+// Untyped blanks are not allowed in the argument that defines the row scope
+>> Filter(Blank(), Blank())
```

Comment:
>Filter(Blank(), Blank()) [](http://example.com/codeflow?start=3&length=24)

This is a breaking change: we won't allow untyped blanks to define the row scope. There isn't a real scenario where this would be useful, besides language semantic tests.